Doc/library/functions.rst

```diff
@@ -81,9 +81,23 @@ are always available.  They are listed here in alphabetical order.
 
 .. function:: bin(x)
 
-   Convert an integer number to a binary string. The result is a valid Python
-   expression.  If *x* is not a Python :class:`int` object, it has to define an
-   :meth:`__index__` method that returns an integer.
+   Convert an integer number to a binary string prefixed with "0b". The result is
+   a valid Python expression. If *x* is not a Python :class:`int` object, it has
+   to define an :meth:`__index__` method that returns an integer. Some examples:
+
+      >>> bin(12)
+      '0b1100'
+      >>> bin(-10)
+      '-0b1010'
+
+   If prefix "0b" is desired or not, you can use either of the following ways. See
+   also :func:`format` for more information.
+
+      >>> num = 14
+      >>> format(num, '#b'), format(num, 'b')
```

Comment:
I suggest to remove the num variable and just write 14 directly, to simplify examples (and easier to copy/paste).